### PR TITLE
Inject informations into about dialog

### PR DESCRIPTION
**Pull request by Tristan-WorkGH**
Dependent of:
* gridsuite/deployment#353
* gridsuite/study-server#484
* gridsuite/commons-ui#321

### CODE REVIEW

File: src/components/app-top-bar.js [hunk 1]
```diff
@@ -81,26 +87,6 @@ const AppTopBar = ({ user, userManager }) => {
...
-    }
-
-    function onLogoClicked() {
-        navigate('/', { replace: true });
-    }
-
-    const showPopupConfigurationProcesses = () => {
-        setShowConfigurationProcesses(true);
-    };
-
```
Comment (EstherDarkish) [inline src/components/app-top-bar.js:103]: These do not belong to this PR

File: package.json [hunk 2]
```diff
@@ -23,6 +24,7 @@
         "prop-types": "^15.7.2",
         "react": "^18.0.0",
         "react-dom": "^18.0.0",
+        "react-hook-form": "^7.48.2",
```
Comment (flomillot) [inline package.json]: I think you can remove yup and rhf it was a mistake

Comment (Tristan-WorkGH) [reply package.json]: dunno, they were added in last 2 commits, I don't know if they're mandatory or not.

File: src/components/app-top-bar.js (hunk 1 above)
Comment (Tristan-WorkGH) [reply src/components/app-top-bar.js:103]: I did like in gridstudy-app PR

File: package.json (hunk 2 above)
Comment (flomillot) [reply package.json]: They are not used in the project. Don't include them in the `dependencies`.

Comment (flomillot) [reply package.json]: Ok apparently we can't compile without because it's in peer deps of commons ui, so just let it in dev deps

**Review by flomillot — APPROVED** [review]

**Review by EstherDarkish — APPROVED** [review]
